src/debugAdapter/transaction/transactionProvider.ts

```diff
@@ -26,25 +27,39 @@ export class TransactionProvider {
   }
 
   public async getLastTransactionHashes(take: number = TRANSACTION_NUMBER_TO_SHOW): Promise<string[]> {
-    let latestBlockNumber: any;
+    let latestBlock: any;
```

Comment:
- Would it be possible to type this as `number`?
- And given that contains the block number, should we keep the `Number` suffix, just to be more explicit? It also would help us to differentiate between block numbers and blocks.

Reply:
Ok, no problems